src/Input/KeyInterop.ts

```diff
@@ -170,8 +170,9 @@ module Fayde.Input {
             }
             return args;
         }
+		
         CreateArgsDown(e): Fayde.Input.KeyEventArgs {
-            if (e["char"] && e.keyCode !== 8 && e.keyCode !== 9)
+            if (e["char"] && e.keyCode !== 8 && e.keyCode !== 9 && !e.ctrlKey)

```

Comment:
The functionality is to do Control-c, control-v, control-x for Copy, paste and cut. Then there is Control-left, right, home, end for movement of the carret. There will be others in the future as well. Control-b, u, i, etc... for bold, underline, italics, etc...